### PR TITLE
Organize Repo

### PR DESCRIPTION
**Pull request by MattsonCam**
In this pr, I update the readmes and remove some unnecessary components to improve overall organization for the phenotypic profiling paper. Please let me know if you have any suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,9 @@
...
 
-In this dataset, there are 51 plates that are associated with one of three treatment types:
+In this dataset, there are 51 plates with one of three perturbation types (Clustered Regularly Interspaced Short Palindromic Repeats [CRISPR], Open Reading Frame [ORF], and Compound) for two cell lines (A549 and U2OS).
 
-- Compound
-- CRISPR
-- ORF
-
-Each treatment has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
+Each perturbation type has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
```
Comment (gwaybio) [inline README.md:9]:
```suggestion
Each perturbation type has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
We accessed this reference data from the JUMP `experimental-metadata.tsv` file located [here](https://github.com/jump-cellpainting/2024_Chandrasekaran_NatureMethods/blob/a18fd7719c05b638c731142b0d42a92c645e2b33/benchmark/output/experiment-metadata.tsv).
```

Comment (gwaybio) [reply README.md:9]: True?

Comment (gwaybio) [reply README.md:9]: NVM! I see that you added this detail in the specific section's README. Feel free to ignore this

**Review by gwaybio — APPROVED** [review]
🎉

**Comment by MattsonCam** [comment]
Thanks Greg, merging now!

**Pull request merged**